src/utils/kurl-yaml.js

```diff
@@ -0,0 +1,26 @@
+
+exports.openebsComment = `# OpenEBS is the default PVC provisioner, and
```

Comment:
I think we mean to say that OpenEBS will be the default _PV provisioner_ instead of PVC provisioner.

Reply:
or we can say both: `PV/PVC provisioner`